### PR TITLE
Optimise running iOS mocked tests

### PR DESCRIPTION
**Pull request by Bindu-Maharudrappa**

### CODE REVIEW

**Comment by graduad** [comment]
The pattern looks good to me. But perhaps we could use groups as well?

**Comment by Bindu-Maharudrappa** [comment]
Groups make sense, I will update it. Thank you.

File: test_app/integration_tests/mocked/video_presentation_service_test.dart [hunk 1]
```diff
@@ -5,91 +5,93 @@ import 'package:dolbyio_comms_sdk_flutter/dolbyio_comms_sdk_flutter.dart';
...
+      await dolbyioCommsSdkFlutterPlugin.videoPresentation.play();
+      var statePlay =
+          await dolbyioCommsSdkFlutterPlugin.videoPresentation.state();
+      expect(statePlay, VideoPresentationState.play);
 
       await expectNative(
           methodChannel: videoPresentationServiceAssertsMethodChannel,
           assertLabel: "assertStateArgs",
-          expected: {"hasRun": true, "state": 0});
+          expected: {"hasRun": true, "state": 1});
```
Comment (kbetl-dlb) [inline test_app/integration_tests/mocked/video_presentation_service_test.dart:68]: Why expected state was change from 0 to 1 ?

Comment (Bindu-Maharudrappa) [reply test_app/integration_tests/mocked/video_presentation_service_test.dart:68]: Oh sorry, I didn't change the state. The code is rearranged I suppose. For the play() method the state was 1 and for the stop() method the state was 0.
The state you’re looking for is in the below part of the code I think.

Comment (kbetl-dlb) [reply test_app/integration_tests/mocked/video_presentation_service_test.dart:68]: ok, my mistake. Sometimes git show changes in very weired way :)

Comment (Bindu-Maharudrappa) [reply test_app/integration_tests/mocked/video_presentation_service_test.dart:68]: That's true, I agree with that :)

**Pull request merged**